code/modules/mob/living/carbon/human/inventory.dm

```diff
@@ -71,18 +71,26 @@
 			return has_limb("chest")
 		if(WEAR_ACCESSORY)
 			return has_limb("chest")
+		if(EQUIP_IN_BOOT)
+			return has_limb("r_foot") && has_limb("l_foot")
 		if(WEAR_IN_BACK)
-			return TRUE
+			return has_limb("chest")
```

Comment:
im pretty sure you cant remove your chest

Reply:
I'm just replicating already existing checks. The purpose I can see is for re-using the proc with modified species that descend from humans (or different if the proc is changed to `living` or `mob`.

Reply:
I suppose, I feel like it's a bit of a waste of cpu use but I suppose there is always modularity to think of.